### PR TITLE
#827: Fix GitHub Action Runner for Windows

### PR DESCRIPTION
**Pull request by alfeilex**

### CODE REVIEW

File: scripts/pom.xml [hunk 1]
```diff
@@ -129,40 +129,24 @@
...
-  <profiles>
-    <profile>
-      <id>integration-test</id>
-      <activation>
-        <os>
-          <family>!Windows</family>
-        </os>
-        <property>
-          <name>!skipTests</name>
-        </property>
```
Comment (hohwille) [inline scripts/pom.xml:143]: Please do not remove this feature.
When using maven it is a common standard that you can disable the tests using `-DskiptTests=true`.
With your change, you removed this feature.
Please keep the plugin execution in the profile and just remove`<os>...</os>`.

Also please test the build locally on windows (`mvn clean install`):
* Does it work and find `bash` automatically?
* Does it work if you have installed shellcheck locally on windows?

**Review by hohwille — CHANGES_REQUESTED** [review]
@alfeilex Seems you found a working solution that looks good to me. 👍 
However, as commented in my review, please keep the profile to let users disable the tests if needed.
If skippable and locally tested this should be ready for merge.

File: scripts/pom.xml (hunk 1 above)
Comment (alfeilex) [reply scripts/pom.xml]: I have added the `!skipTests` feature again. The build runs locally on windows and find `bash` automatically. The `shellcheck` also works. (I used chocolatey to install shellcheck)

**Review by hohwille — APPROVED** [review]
@alfeilex thanks for the rework. Now looking perfect. Ready for merge 👍

**Pull request merged**